angular-workspace/projects/ni/nimble-angular/src/directives/anchor/tests/nimble-anchor-router-link-with-href.directive.spec.ts

```diff
@@ -98,4 +105,8 @@ describe('Nimble anchor RouterLinkWithHrefDirective', () => {
             expect(routerNavigateByUrlSpy).not.toHaveBeenCalled();
         }));
     });
+
+    it('sanitizes href', () => {
+        expect(sanitizer.sanitize).toHaveBeenCalledWith(SecurityContext.URL, '/page1?param1=true');
+    });
```

Comment:
Are these tests requiring that other tests in the file have already run?

Reply:
No, this test doesn't have a dependency on the other tests in the file. The sanitization happens as part of the component being run the and the directive bindings being process, which all happens in the `beforeEach` of the tests.

Reply:
Maybe the test name could be `sanitized initial bound href`
The active voice of sanitizes makes me expect something to happen in the test